Fix accessibility issues, and a few content tweaks

latrani opened the pull request:
Accessibility issues found via http://wave.webaim.org:

* Improved text contrast in several places
* Fixed some duplicate alt tags
* Fixed a heading hierarchy bug
* Labelled a form element

Other changes:

* Removed the color swatches, they were flagged for contrast and looked
  out of place to me. If these were supposed to be part of the press-
  kit stuff, we should either put them in those files, or add some text
  on the page explaining why they're there.
* Added a (temporary) link back to TheKinrar's instance wizard, to be
  removed once this one has feature parity (Since right now not having
  a link there looks like a regression of features from the user POV)

Gargron commented:
I'm gonna have to take a look at how this look first. Straight away, I don't like removing the color swatches and believe the link to the wizard should be on the left side across from the search input. I am very finnicky with design.

latrani commented:
I'm fine with keeping the color swatches, as long as we also add some text to explain why they are there. :)

As far as the wizard link being on the left across from the search input, I'm all for that; that was my first idea, but I thought it might be too intrusive.

If you give me some explanatory text for the swatches (or just a general idea of why you have them there so I can write some) I can make those changes and resubmit.

The only other visual changes was I bumped the colors up a step (darker -> lighter, lighter -> lightest) where the accessibility tool said we had a readability issue. I'm open to doing more elaborate stuff there too but that seemed like the way that would keep things looking the most cohesive.

latrani commented:
Newest commit addresses the comments above. Hoping we can resolve this soon since we recently got another complaint about the contrast issue on the Discourse board.
